versions.sh

```diff
@@ -90,6 +91,14 @@ for version in "${versions[@]}"; do
 				if [ "$variant" = 'apache' ]; then
 					continue
 				fi
+				if [[ "$rcVersion" = '8.1' ]] && [[ "$suite" = 'alpine3.23' ]]; then
+					# Keep PHP 8.1 with Alpine 3.21 default until end of year; see also https://github.com/docker-library/php/blob/9ab2e4b37addffaa10f06d9e5f54f7bd1f5ef18f/generate-stackbrew-library.sh#L120
```

Comment:
While reviewing, I got confused by this wording because I had forgotten about

https://github.com/docker-library/php/blob/9ab2e4b37addffaa10f06d9e5f54f7bd1f5ef18f/generate-stackbrew-library.sh#L120-L123

(leaving a link here so it's obvious to others too :joy:)

Reply:
Yeah it was a bit of a quick hack and wording, but with the link it should suffice for code that will only be around for a handful of weeks 😉